src/main/java/calculator/Operator.java

```diff
@@ -0,0 +1,47 @@
+package calculator;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public class Operator<T> {
+    @RequiredArgsConstructor
+    public enum Sign{
+        ADD("+"),
+        SUBTRACT("-"),
+        MULTIPLY("*"),
+        DIVIDE("/");
+
+        private final String sign;
+
+        public static Sign fromSignStr(String sign){
+            for (Sign value : Sign.values()) {
+                if(value.sign.equals(sign)) return value;
+            }
+            throw new IllegalArgumentException();
```

Comment:
```suggestion
            return Arrays.stream(values())
                .filter(s -> s.sign.equals(sign))
                .findFirst()
                .orElseThrow(() -> new IllegalArgumentException("잘못된 연산자입니다."));
```

return과 throw 라는 명령문을 통해 메서드에서 나가는 문이 두 군데가 있다면  stream으로 한번에 처리할 수 있지 않을까 합니다.

Reply:
변경하였습니다.